docs/source/get_started/index.rst

```diff
@@ -5,11 +5,55 @@
 First things first
 ******************
 
-With a fresh AiiDA installation, there are still a few steps to complete, before any calculations can be started:
+With ``aiida-core`` installed, a few steps remain before starting to run calculations:
 
-1) The daemon has to be configured and started
-2) A computer has to be configured
-3) A code has to configured
-4) A plugin has to be installed
+1) :ref:`plugins`
+2) :ref:`setup_computer`
+3) :ref:`setup_code`
+
+.. _plugins:
+
+***************
+Install Plugins
+***************
+
+While the ``aiida-core`` package provides the workflow engine and database model, it relies on *plugins* for connecting to specific simulation codes.
+
+Find your AiiDA plugins on the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry>`_. 
```

Comment:
What if the plugin they need doesn't exist? Should we put a link to the documentation for creating a plugin and the `aiida-plugin-cutter`?

Reply:
good point - it's exactly the right point for a link to the plugin development section